pkgs/by-name/so/sou/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  nix-update-script,
+  versionCheckHook,
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "sou";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "knqyf263";
+    repo = "sou";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-uGYCmW60OvEfserujQMXC9r8S8W+EN+w9EXUGjk1vtw=";
+  };
+
+  vendorHash = "sha256-6kgiZx/g1PA7R50z7noG+ql+S9wSgTuVTkY5DIqeJHY=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X=main.version=${finalAttrs.version}"
+  ];
+
+  doInstallCheck = true;
+  nativeInstallCheck = [ versionCheckHook ];
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    description = "Tool for exploring files in container image layers";
+    homepage = "https://github.com/knqyf263/sou";
```

Comment:
```
changelog = "https://github.com/knqyf263/sou/releases/tag/v${finalAttrs.version}";
```
We shoudl add the changelog since the project publishes it